index.js

```diff
@@ -3,53 +3,227 @@ const state = {
     {
       id: "001-beetroot",
       name: "beetroot",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "002-carrot",
       name: "carrot",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "003-apple",
       name: "apple",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "004-apricot",
       name: "apricot",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "005-avocado",
       name: "avocado",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "006-bananas",
       name: "bananas",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "007-bell-pepper",
       name: "bell pepper",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "008-berry",
       name: "berry",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "009-blueberry",
       name: "blueberry",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "010-eggplant",
       name: "eggplant",
-      price: 0.35
-    }
+      price: 0.35,
+    },
   ],
-  cart: []
-};
\ No newline at end of file
+  cart: [],
+};
+
+// storeItems, cartItems and total selectors
+const storeItemsUl = document.querySelector(".store--item-list");
+const cartItemsUl = document.querySelector(".cart--item-list");
+
+// clear the stroreItems and cartItems
+function clearingStoreItems() {
+  storeItemsUl.innerText = "";
+}
+
+function clearingCart() {
+  cartItemsUl.innerText = "";
+}
+
+function renderingItemsToStore() {
+  clearingStoreItems();
+  // Create the for loop for the items in the store
+  for (let i = 0; i < state.items.length; i++) {
+    const specificFruit = state.items[i];
+
+    // section 1: create the list item like template
+    // create the list items by createelement
+    // than append li to ul
+    const storeItemsList = document.createElement("li");
+    storeItemsUl.append(storeItemsList);
+
+    // create the div
+    // set an attribute class to the div
+    const itemsDiv = document.createElement("div");
+    itemsDiv.setAttribute("class", "store--item-icon");
```

Comment:
instead of using setAttribute for everything you could use .className = , or .src =